corehq/apps/app_manager/views/utils.py

```diff
@@ -373,19 +374,34 @@ def update_linked_app(app, user_id):
                 ).format(ucr_id=str(e))
             )
 
-    if app.master_is_remote:
-        try:
-            pull_missing_multimedia_for_app(app)
-        except RemoteRequestError:
-            raise AppLinkError(_(
-                'Error fetching multimedia from remote server. Please try again later.'
-            ))
-
     app.domain_link.update_last_pull('app', user_id, model_details=AppLinkDetail(app_id=app._id))
 
     # reapply linked application specific data
     app.reapply_overrides()
 
+    if app.master_is_remote:
+        pull_missing_multimedia_for_app.delay(app.domain, app.get_id)
```

Comment:
Ah, I think I missed this piece earlier.  Is the idea here that the "pull multimedia" phase _is_ done as part of the build, but it's changed to be non-blocking, though the user will be forced to let it succeed before making a build?

Reply:
> "pull multimedia" phase is done as part of the build
asynchronously.

> though the user will be forced to let it succeed before making a build
correct